README.md

```diff
@@ -17,6 +17,17 @@ With multiple breathing patterns, Nafas helps you find your way to a detoxified
 No need to walk away to take a break, just sit comfortably, run Nafas and let the journey begin.
 **Nafas** means breath in Persian.
 
+Nafas offers a selection of predefined breathing exercise programs.
+Each program consists of multiple cycles.
+The exercises begin with a gentle preparation phase to help users settle in and focus, followed by a series of timed inhales and exhales.
+Between these breaths, the programs incorporate deliberate pauses that allow users to retain and sustain their breath.
+These cycles aim to enhance both physical and mental well-being.
+
+<div align="center">
+	<img src="https://github.com/sepandhaghighi/nafas/raw/master/otherfiles/overview.png" width=100px>
+	<p>Nafas Programs' Cycle. Each program starts with a preparation phase and goes through several inhales and exhalations with pauses as it retains and sustains.</p>
```

Comment:
Too long:

<p>Nafas Programs' Cycle<s>. Each program starts with a preparation phase and goes through several inhales and exhalations with pauses as it retains and sustains.</s></p>